src/app/frontend/commons/commons_module.js

```diff
@@ -0,0 +1,11 @@
+import errorDialogService from './errordialog_service';
+
+/**
+ * Angular module containing navigation chrome for the application.
+ */
+export default angular.module(
+                          'kubernetesDashboard.commons',
+                          [
+                            'ngMaterial',
+                          ])
+    .factory('errorDialog', errorDialogService);
```

Comment:
angular/no-service-method

You should prefer the factory(9 method instead of service()